### ISSUE TITLE
How to specify allowUnknown for a schema created via and

### ISSUE THREAD

**Issue by frankie-eric** [davidmdm/myzod#34]
I can specify `{allowUnknown: true}` for a schema created via `myzod.object` method. 

Is there a way I can do the same for schema created via `myzod.and`?

**Comment by davidmdm** [comment]
should either object in the intersection allow unknown, the intersection will allow for unknown keys. However you are right that there is no way to set the allowUnknown property once an ObjectSchema has already been created. 

I will remedy this by adding the fluent syntax:
```
myzod.object({ ... }).allowUnknown()
```

it will be released with myzod 1.5.0 this afternoon.

**Comment by davidmdm** [comment]
@frankie-eric It has been released with myzod 1.5.0

```typescript
const schema1 = z.object({...})
const schema2 = z.object({...})
const schema3 = schema1.and(schema2).allowUnknownKeys();
```

I will leave this open for a little while to get your feedback but otherwise I will close this issue.
Thanks!

**Comment by frankie-eric** [comment]
Many thanks @davidmdm for your super quick response and action. 👍 

BTW I did try something like the following and it was not working, probably because I was using `1.2.2`.

```
const schema1 = z.object({...}, {allowUnknown: true});
const schema2 = z.object({...}, {allowUnknown: true});
const schema3 = schema1.and(schema2);
// schema3.parse throws error when parsing an object contains key which not exists in schema1 and schema2
```
